mithril-aggregator/src/commands/mod.rs

```diff
@@ -101,9 +101,10 @@ impl MainOpts {
     /// get log level from parameters
     pub fn log_level(&self) -> Level {
         match self.verbose {
-            0 => Level::Warning,
-            1 => Level::Info,
-            2 => Level::Debug,
+            0 => Level::Critical,
```

Comment:
It should probably `Error` instead?
```suggestion
            0 => Level::Error,
```